src/main/java/uk/gov/hmcts/darts/common/repository/EventRepository.java

```diff
@@ -103,13 +103,12 @@ having count(e2.event_id) > 1
 
     @Query(value = """
         select distinct on (event_id, hearing_ids) e.* from (
-            SELECT e.eve_id, event_id, string_agg(he.hea_id::varchar, ',' order by he.hea_id) as hearing_ids FROM darts.event e
+            SELECT e.eve_id, event_id, e.created_ts, string_agg(he.hea_id::varchar, ',' order by he.hea_id) as hearing_ids FROM darts.event e
```

Comment:
is it possible to add a test for the scenario covered in the ticket?

Reply:
Added :)